### PR TITLE
transfer documentation of lib.rs to minimal.rs

### PR DESCRIPTION
**Pull request by Keksoj**
As pointed out in 

- #879 

The code samples provided by the documentation in `lib.rs` are obsolete, flagged with `ignore` and difficult to maintain. This is very misguiding.

Since these code samples match exactly `lib/example/minimal.rs`, it seems more appropriate to put the comments there.

### CODE REVIEW

**Comment by FlorentinDUBOIS** [comment]
We should keep both as the description `lib.rs` is read by the documentation generator and displayed.

File: lib/src/lib.rs [hunk 1]
```diff
@@ -7,142 +7,7 @@
 //! reloading from a file regularly. The event loop runs in its own thread
 //! and receives commands through a message queue.
 //!
-//! To create a HTTP proxy, you first need to create a `HttpProxyConfiguration`
```
Comment (FlorentinDUBOIS) [inline lib/src/lib.rs:10]: As mention globally, this should be kept, because it is displayed in the documentation.

Comment (Keksoj) [reply lib/src/lib.rs:10]: Okay, it's done!

**Comment by Keksoj** [comment]
Everything has changed with protobuf, I will start over

**Pull request closed without merging**